widgetastic_manageiq.py

```diff
@@ -878,3 +878,155 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class Stepper(Widget):
+    """ A CFME Stepper Control
+
+    .. code-block:: python
+
+        stepper = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+        stepper.increase()
+    """
+    ROOT = ParametrizedLocator('{@locator}')
+    INPUT_CTL = './/input[contains(@class, "bootstrap-touchspin")]'
+
+    def __init__(self, parent, locator, logger=None):
+        Widget.__init__(self, parent=parent, logger=logger)
+
+        self.locator = locator
+
+    @property
+    def minus_button(self):
+        return Button(self, '-')
```

Comment:
@mfalesni , @RonnyPfannschmidt I get strange errors when define those controls in class

```
Traceback (most recent call last):
  File "/home/ez/proj/integration_tests/.cfme/local/lib/python2.7/site-packages/IPython/core/interactiveshell.py", line 2881, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-7-6c05ea04a9a3>", line 1, in <module>
    stepper.increase()
  File "/home/ez/proj/integration_tests/widgetastic_manageiq.py", line 910, in increase
    self.plus_button.click()
  File "/home/ez/proj/integration_tests/.cfme/local/lib/python2.7/site-packages/widgetastic/widget.py", line 79, in __get__
    if self not in obj._widget_cache:
AttributeError: 'Stepper' object has no attribute '_widget_cache'
```